apps/api/Transcript Summarization.postman_collection.json

```diff
@@ -69,6 +69,53 @@
 				}
 			},
 			"response": []
+		},
```

Comment:
@ramraut-osm  Did you test with `in` operator?
in along with a field and list of values sshould return all such records.
We will be using it with ids but should be generic.

Make sure that example is present in postman